include/certifier_framework.i

```diff
@@ -13,9 +13,19 @@
 %module certifier_framework
 %include "std_string.i"
 
-// Needed to invoke init_policy_key() using Python bytestream arg.
+// Needed to invoke init_policy_key() and other interfaces that take
+// (byte *, int) supplied by Python bytestream arg.
 %include <pybuffer.i>
-%pybuffer_binary(byte* asn1_cert, int asn1_cert_size)   // cc_trust_data()->init_policy_key()
+
+%pybuffer_binary(const byte *serialized_attest_key, int attest_key_size);
+%pybuffer_binary(const byte *measurement, int measurement_size);
+%pybuffer_binary(const byte *serialized_attest_endorsement, int attest_key_signed_claim_size);
+
+%pybuffer_binary(byte* asn1_cert, int asn1_cert_size);   // cc_trust_data()->init_policy_key()
+
+%ignore initialize_simulated_enclave(const string &serialized_attest_key,
```

Comment:
This is being ignored; otherwise, we have over-loaded interfaces. 

In the SWIG-generated wrap.cc file, the logic to pick the right interface is not working as expected. The code-flow gets confused and we get an error that the arguments are incorrect.

By ignoring this interface, we leave just the interface with triplets of byte-stream interfaces as the valid Python interface to invoke. This allows tests to work cleanly.